build.sbt

```
@@ -68,7 +67,7 @@ lazy val root = (project in file("."))
       case x => MergeStrategy.first
     },
     // Build assembly jar, this builds an uberJar with all dependencies
-    assembly / assemblyJarName := s"${name.value}-${version}.jar",
+    assembly / assemblyJarName := s"${name.value}-${version.value}.jar",
```

Should this just be `version` now since we've removed the `val` up above?

```suggestion
    assembly / assemblyJarName := s"${name.value}-${version}.jar",
```

---

Discussed this on slack!